neo/Network/RPC/TransactionManager.cs

```diff
@@ -50,11 +49,12 @@ public TransactionManager(RpcClient rpc, UInt160 sender)
         /// <returns></returns>
         public TransactionManager MakeTransaction(byte[] script, TransactionAttribute[] attributes = null, Cosigner[] cosigners = null, long networkFee = 0)
         {
+            var random = new Random();
```

Comment:
It it good?
Why not keep as before, initializing and let the seed feed the generator?

Reply:
It's safer

Reply:
Can we predict next Random based on historical values? @igormcoelho